shared/references/cce-redhat-avail.txt

```diff
@@ -677,12 +677,3 @@ CCE-82890-5
 CCE-82891-3
 CCE-82892-1
 CCE-82893-9
-CCE-82894-7
-CCE-82895-4
-CCE-82896-2
-CCE-82898-8
-CCE-82899-6
-CCE-82900-2
-CCE-82901-0
-CCE-82902-8
-CCE-82903-6
```

Comment:
@shawndwells why have so many CCEs been removed?

Reply:
Three rules with three CCEs each.

Reply:
hahahahaha.... more coffee is needed for me!